README.md

```diff
@@ -39,13 +39,14 @@ Then, enable the formatter in your `config.exs`:
 
 ```elixir
 config :logger, :default_handler,
-  formatter: {LoggerJSON.Formatters.Basic, []}
+  formatter: LoggerJSON.Formatters.Basic.new(metadata: [:request_id])
 ```
 
 or during runtime (eg. in your `application.ex`):
 
 ```elixir
-:logger.update_handler_config(:default, :formatter, {Basic, []})
```

Comment:
I think aliased names shouldn't be used in READMEs and code examples :)

Even knowing how it al works, it took me a few seconds to realize the problem here:

```elixir
iex(1)> :logger.update_handler_config(:default, :formatter, {Basic, []})
:ok
iex(2)> require Logger
Logger
iex(3)> Logger.info("hey")
2024-11-27T13:13:26.074101+07:00 info: FORMATTER CRASH: {string,<<"hey">>}
:ok
2024-11-27 13:13:26.080851 {error,simple_handler_process_dead}
2024-11-27 13:13:26.080703
                           2024-11-27T13:13:26.074545+07:00 debug: FORMATTER CRASH: {report,[{formatter_crashed,'Elixir.Basic'},{config,[]},{log_event,#{meta => #{pid => <0.3083.0>,time => 1732688006074101,gl => <0.70.0>,domain => [elixir]},msg => {string,<<"hey">>},level => info}},{reason,{error,undef,[{'Elixir.Basic',format,[#{meta => #{pid => <0.3083.0>,time => 1732688006074101,gl => <0.70.0>,domain => [elixir]},msg => {string,<<"hey">>},level => info},[]],[]}]}}]}
    formatter_crashed: 'Elixir.iex(4)> Basic'
    config: []
    log_event: #{meta=>#{line=>411,pid=><0.3083.0>,time=>1732688006074545,file=>"logger_h_common.erl",gl=><0.70.0>,report_cb=>fun logger:format_otp_report/1,mfa=>{logger_h_common,try_format,3},internal_log_event=>true},msg=>{report,[{formatter_crashed,'Elixir.Basic'},{config,[]},{log_event,#{meta=>#{pid=><0.3083.0>,time=>1732688006074101,gl=><0.70.0>,domain=>[elixir]},msg=>{string,<<"hey">>},level=>info}},{reason,{error,undef,[{'Elixir.Basic',format,[#{meta=>#{pid=><0.3083.0>,time=>1732688006074101,gl=><0.70.0>,domain=>[elixir]},msg=>{string,<<"hey">>},level=>info},[]],[]}]}}]},level=>debug}
    reason: {error,undef,[{'Elixir.Basic',format,[#{meta=>#{line=>411,pid=><0.3083.0>,time=>1732688006074545,file=>"logger_h_common.erl",gl=><0.70.0>,report_cb=>fun logger:format_otp_report/1,mfa=>{logger_h_common,try_format,3},internal_log_event=>true},msg=>{report,[{formatter_crashed,'Elixir.Basic'},{config,[]},{log_event,#{meta=>#{pid=><0.3083.0>,time=>1732688006074101,gl=><0.70.0>,domain=>[elixir]},msg=>{string,<<"hey">>},level=>info}},{reason,{error,undef,[{'Elixir.Basic',format,[#{meta=>#{pid=><0.3083.0>,time=>1732688006074101,gl=><0.70.0>,domain=>[elixir]},msg=>{string,<<"hey">>},level=>info},[]],[]}]}}]},level=>debug},[]],[]}]}
```

Reply:
Related: https://github.com/Nebo15/logger_json/pull/143